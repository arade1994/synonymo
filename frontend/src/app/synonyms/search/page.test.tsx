import { describe, expect, test } from "vitest";

import { render, screen } from "@testing-library/react";

import SearchSynonymsPage from "./page";

describe("SearchSynonymsPage", () => {
  test("It should render and match a snapshot by default", () => {
    const { baseElement } = render(<SearchSynonymsPage />);

    expect(baseElement).toMatchSnapshot();
  });

  test("It should render the home link", () => {
    render(<SearchSynonymsPage />);

    const homeLink = screen.getByTestId("homeLink");

    expect(homeLink).toBeDefined();
  });
});
