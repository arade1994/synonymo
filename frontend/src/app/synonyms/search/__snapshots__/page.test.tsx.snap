// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SearchSynonymsPage > It should render and match a snapshot by default 1`] = `
<body>
  <div>
    <main
      class="_main_8812a5"
    >
      <a
        class="_homeLink_8812a5"
        href="/"
        id="homeLink"
      >
        ‚Üê Back to Home
      </a>
      <h1
        class="_title_8812a5"
      >
        Search for Synonyms
      </h1>
      <p>
        Search functionality coming soon!
      </p>
    </main>
  </div>
</body>
`;
